lambdas/tests/unit/handlers/test_search_patient_details_handler.py

```diff
@@ -18,126 +17,77 @@ def patch_env_vars():
     with patch.dict(os.environ, env_vars):
         yield env_vars
 
+@pytest.fixture()
+def mocked_context(mocker):
+    mocked_context = mocker.MagicMock()
+    mocked_context.authorization = {
+        "selected_organisation": {"org_ods_code": "Y12345"},
+        "repository_role": "GP_ADMIN",
+    }
+    _ = mocker.patch(
+        "handlers.search_patient_details_handler.request_context", mocked_context
+    )
+    yield _
 
 def test_lambda_handler_valid_id_returns_200(
-    valid_id_event_with_auth_header, context, mocker, patch_env_vars
+    valid_id_event_with_auth_header, context, mocker, mocked_context
 ):
-    response = Response()
-    response.status_code = 200
-    response._content = json.dumps(PDS_PATIENT).encode("utf-8")
+    patient_details = """{"givenName":["Jane"],"familyName":"Smith","birthDate":"2010-10-22",
+        "postalCode":"LS1 6AE","nhsNumber":"9000000009","superseded":false,
+        "restricted":false,"generalPracticeOds":"Y12345","active":true}"""
```

Comment:
Just a trivial thing, this json string of patient detail probably is the same as the one at `test_search_patient_details_service.py` line 82, so I think we might put this string in conftest file so that its easier to reuse in the future.

But its up to you and I am happy to keep it as it is now :)